openedx/features/content_tagging/rest_api/v1/tests/test_views.py

```diff
@@ -0,0 +1,180 @@
+"""
+Tests tagging rest api views
+"""
+
+import ddt
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from openedx.features.content_tagging.models import Taxonomy
+
+User = get_user_model()
+
+
+@ddt.ddt
+class TestTaxonomyViewSet(APITestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.user = User.objects.create(
+            username="user",
+            email="user@example.com",
+        )
+
+        self.staff = User.objects.create(
+            username="staff",
+            email="staff@example.com",
+            is_staff=True,
+        )
+
+    @ddt.data(
+        (None, status.HTTP_403_FORBIDDEN),
+        ("user", status.HTTP_403_FORBIDDEN),
+        ("staff", status.HTTP_200_OK),
+    )
+    @ddt.unpack
+    def test_list_taxonomy(self, user_attr, expected_status):
+        url = reverse("oel_tagging:taxonomy-list")
+
+        user = getattr(self, user_attr) if user_attr else None
+        if user:
+            self.client.force_authenticate(user=user)
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, expected_status)
+
+    @ddt.data(
+        (None, {"enabled": True}, status.HTTP_403_FORBIDDEN),
+        ("user", {"enabled": True}, status.HTTP_200_OK),
+        ("user", {"enabled": False}, status.HTTP_404_NOT_FOUND),
+        ("staff", {"enabled": True}, status.HTTP_200_OK),
+        ("staff", {"enabled": True}, status.HTTP_200_OK),
+    )
+    @ddt.unpack
+    def test_detail_taxonomy(self, user_attr, taxonomy_data, expected_status):
+        create_data = {**{"name": "taxonomy detail test"}, **taxonomy_data}
+        taxonomy = Taxonomy.objects.create(**create_data)
+        url = reverse("taxonomy-detail",
```

Comment:
Please ensure that all of the URLs used here are prefixed with the `content_tagging:` namespace (and that this works with your URL setup).